contrib/pkg/createcluster/create.go

```diff
@@ -511,6 +513,7 @@ func (o *Options) GenerateObjects() ([]runtime.Object, error) {
 		BaseDomain:            o.BaseDomain,
 		ManageDNS:             o.ManageDNS,
 		DeleteAfter:           o.DeleteAfter,
+		HibernateAfter:        o.HibernateAfter,
```

Comment:
It seems like a better interface for the builder if it takes a `*time.Duration` instead of a `string` for `HibernateAfter`. The create-cluster `Complete` function could translate the string to a duration so that the builder can take a duration.

Reply:
Agreed, changed to dur.